runtime/compiler/env/VMJ9.h

```diff
@@ -279,6 +279,16 @@ class TR_J9VMBase : public TR_FrontEnd
    virtual bool jitStaticsAreSame(TR_ResolvedMethod *, int32_t, TR_ResolvedMethod *, int32_t);
    virtual bool jitFieldsAreSame(TR_ResolvedMethod *, int32_t, TR_ResolvedMethod *, int32_t, int32_t);
 
+   /**
+    * \brief Check is object is an array class instance
+    *
+    * \param currentThread the current thread
+    * \param object the object address
+    * \return true if object is an array class instance
+    * \return false if object is not an array class instance
+    */
+   virtual bool objectIsArray(J9VMThread *currentThread, TR_OpaqueClassBlock *object);
```

Comment:
You may have chosen this name because it matches the helper you end up calling, but since this is an inquiry API in the FrontEnd would it be better to call it `isArrayObject()` instead?